views/js/randomColorsForScores.js

```diff
@@ -0,0 +1,5 @@
+window.onload = function () {
+  document.querySelectorAll("#high_scores_container .individual_score").forEach(function (element) {
+    element.style.color = Math.floor(Math.random() * 16777215).toString(16);
```

Comment:
Some of the random colors can be a bit unreadable against the background (dark purple in your [screenshot](https://screenshot.googleplex.com/r3kWwkaDSeb)), on top of possibly clashing with each other. If you could constrain the range to bright colors only it might work better, not sure if it would be a good use of time though.

Reply:
One other option here is to have a palette of e.g. 16 colors that look good on the background, and randomly select from them.

This might align with the 8-bit aesthetic.

Reply:
That could work. The random color generator just generates all possible integers in the color space, so I can simply proportionally scale it in order to add lightness to the darker colors and make everything more readable. That should be a 2 line fix.

Reply:
Done https://github.com/knative-portability/Kubercade/pull/17/commits/3fd29be8747d2ab7232197a3afe942dc4bbff8d5